File: lua/autorun/simfphys_hitbox_init.lua
version_MAJOR = 1
version_MINOR = 0
version_PATCH = 0
version_TAG = "alpha"
print(
    (((((("simphys hitboxes " .. tostring(version_MAJOR)) .. ".") .. tostring(version_MINOR)) .. ".") .. tostring(version_PATCH)) .. "-") .. tostring(version_TAG)
)
AddCSLuaFile()
function addCSLuaFile(filename)
    if SERVER then
        AddCSLuaFile(filename)
    else
        include(filename)
    end
end
addCSLuaFile("hitbox/cl_hitbox_renderer.lua")
if SERVER then
    include("hitbox/sv_networking.lua")
    include("hitbox/sv_hitbox.lua")
end

File: src/autorun/simfphys_hitbox_init.ts
const version_MAJOR = 1
const version_MINOR = 0
const version_PATCH = 0
const version_TAG = "alpha"


print("simphys hitboxes " + version_MAJOR + "." + version_MINOR + "." + version_PATCH + "-" + version_TAG)

AddCSLuaFile()

function addCSLuaFile(this: void, filename: string) {
	if (SERVER) {
		AddCSLuaFile(filename)
	} else {
		include(filename)
	}
}
addCSLuaFile("hitbox/cl_hitbox_renderer.lua")
if (SERVER) {
	include("hitbox/sv_networking.lua")
	include("hitbox/sv_hitbox.lua")
}
